Usiqee/Module/Global/Views/MSegmentedMenu/MSegmentedMenu.swift

```diff
@@ -22,7 +22,9 @@ class MSegmentedMenu: UIView {
     private var contentTexts: [String] = []
     private var tabs: [MSegmentedMenuItem] = []
     weak var delegate: MSegmentedMenuDelegate?
-    
+    var selectedItem: Int {
+        currentSelectedItem
+    }
```

Comment:
Maybe we can combine selectedItem  and currentSelectedItem by defined private set and public get ?